src/lamp_py/migrations/versions/performance_manager_prod/004_896dedd8a4db_dwell_time_update.py

```diff
@@ -0,0 +1,192 @@
+"""dwell_time_update
+
+Revision ID: 896dedd8a4db
+Revises: 2dfbde5ec151
+Create Date: 2024-07-03 08:53:01.675273
+
+This change simplifies dwell_time calculations to not perform dwell time calculations
+for stations at the start or end of a trip
+
+Details
+* upgrade -> iterate through service_date/static_version_key combos and update dwell_time
+
+* downgrade -> Nothing
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+from lamp_py.runtime_utils.process_logger import ProcessLogger
+
+# revision identifiers, used by Alembic.
+revision = "896dedd8a4db"
+down_revision = "2dfbde5ec151"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    date_query = sa.text(
+        "SELECT DISTINCT service_date, static_version_key FROM vehicle_trips ORDER BY service_date"
+    )
+
+    conn = op.get_bind()
+    result = conn.execute(date_query)
+    for service_date, static_version_key in result.fetchall():
+        stop_sync_update = f"""
```

Comment:
this update looks like its recalculating the dwell times for each vehicle event? i guess we haven't deployed to dev or staging yet, but i would think that its going to take a while?